demisto_sdk/commands/common/docker_helper.py

```diff
@@ -440,6 +440,7 @@ def get_mounts(files: FILES_SRC_TARGET) -> List[Mount]:
                 logger.debug(f"Failed to mount {src} to {target}")
         return mounts
 
+    @retry(times=3)
```

Comment:
do we want to catch any kind of exception here? if no, define which exceptions should be caught to trigger the retry